contributing/development/core_and_modules/binding_to_external_libraries.rst

```diff
@@ -172,7 +172,7 @@ environment's paths:
     # This is a path relative to /modules/tts/ where your .a libraries reside.
     # If you are compiling the module externally (not in the godot source tree),
     # these will need to be full paths.
```

Comment:
This comment should be changed as well, since it still mentions that paths must be relative. Something like this:
```suggestion
    # This is an absolute path where your .a libraries reside.
    # If using a relative path, you must convert it to a
    # full path using an utility function, such as `Dir('...').abspath`.
```